content/participate/report-issue.adoc

```diff
@@ -88,10 +88,8 @@ https://issues.jenkins.io/secure/Dashboard.jspa[the JIRA home page].
 
 * For *Project* select:
 ** _Jenkins_ for general issues with Jenkins,
-** _Security Issues_ if you want to report a security issue privately,
-or
-** _Infrastructure_ if you're reporting a bug with the Jenkins web site,
-wiki, or any other services run by the Jenkins project.
+** _Security Issues_ if you want to report a security issue privately
+* For _Infrastructure_, if you're reporting a bug with the Jenkins web site or any other services run by the Jenkins project, link: open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].
```

Comment:
I wouldn't mention the website as 99% of issues belong to the website and not infra.

```suggestion
* For _Infrastructure_, if you're reporting a bug with a Jenkins service run by the Jenkins project, link: open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].
```